Sources/Swift/Metrics/CounterMetric.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+class CounterMetric: Metric {
```

Comment:
I only added the CounterMetric for now. I will add gauge, distribution and set in other PRs.